easy_phi/tests/auth_test.py

```diff
@@ -112,3 +114,198 @@ def test_no_passwd_required(self):
                          "Dummy authorization backend did not redirect without "
                          "password. Response status: {0}".format(
                              response.code))
+
+
+@unittest.skipIf('easy_phi.auth.PasswordAuthLoginHandler'
```

Comment:
that awkward moment when you realize unit test is bigger than feature itself.. actually, it was very useful and already prevented couple bugs that would be very hard to find otherwise